crates/eips/src/eip4895.rs

```diff
@@ -57,32 +57,32 @@ impl Withdrawal {
 )]
 #[cfg_attr(any(test, feature = "arbitrary"), derive(arbitrary::Arbitrary))]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
-pub struct Withdrawals(Vec<Withdrawal>);
+pub struct Withdrawals(pub Vec<Withdrawal>);
```

Comment:
I personally think this makes the API a bit confused, either we make this `pub` and remove the `iter`/`into_inner` or we continue as the API is today and add `as_ref`. there's no reason to have eg `iter` if this is `pub` as you can now just do `withdrawals.0.iter()`

Reply:
I agree and I think that we can remove iter/into_inner, will make everything more generic with less functions. Let me know what you think and I can adapt

Reply:
actually we already have `AsRef` for this (through derive), so I believe we can use it in reth and avoid any changes here?